### PR TITLE
1854 improve example add assessor switching

### LINKED ISSUE #1854: Improve example - add assessors
Please add functionality to switch between the following assessors in the example:
– Standard SEO assessor (currently implemented).
– Cornerstone SEO assessor.
– Related keywords assessor
– Taxonomy SEO assessor.
– Taxonomy related keywords assessor.

Requires #1826

### PR DESCRIPTION
**Pull request by hansjovis**
## Summary

**Note**: this branch was branched from the #1826 branch. #1826 should be merged first before merging this branch into develop.

This PR can be summarized in the following changelog entry:

* Adds toggles to use the different assessors in the webpack example. E.g. for all the different combinations for cornerstone and taxonomy pages and related keyphrase.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Startup the webpack example.
   * I.o.w. go to `examples/webpack` in the console and enter the `yarn start` command.
* Test out all the different combinations of the `Is a related keyphrase`, `Is cornerstone article` and `is taxonomy page`.
   * For each combination, check whether all the assessments that are defined for this assessor are run (see [this wiki page](https://github.com/Yoast/YoastSEO.js/wiki/SEO-Assessors) for a list).
   * For each combination, (sanity) check whether all the assessments that are run provide the expected results.

Fixes #1854

### CODE REVIEW

**Comment by igorschoester** [comment]
Interesting points.
1 & 2. This could be nice when in a wide resolution. If combined like this and the resolution is less wide, move the toggles to under the other inputs?
4. Is this an alternative to your first and second point? How does this work in a < 768 wide resolution?
5. It is meant to test the functionality of the swapping. Since we use `YoastSEO.js` with and without the `Keyphrase Distribution` assessment.

File: src/worker/AnalysisWebWorker.js [hunk 1]
```diff
@@ -687,6 +687,8 @@ export default class AnalysisWebWorker {
 					synonyms: this._relatedKeywords[ key ].synonyms,
 				} );
 
+				console.log( this._relatedKeywords[ key ] );
```
Comment (igorschoester) [inline src/worker/AnalysisWebWorker.js]:
```suggestion
```

File: src/worker/AnalysisWebWorker.js [hunk 2]
```diff
@@ -687,6 +687,8 @@ export default class AnalysisWebWorker {
...
+
```
Comment (igorschoester) [inline src/worker/AnalysisWebWorker.js]:
```suggestion
```

File: examples/webpack/src/redux/utils/StoreSubscriber.js [hunk 3]
```diff
@@ -46,18 +58,26 @@ export default class StoreSubscriber {
...
 		const { configuration } = state;
 
-		if ( get( prevConfiguration, [ "useKeywordDistribution" ] ) !== get( configuration, "useKeywordDistribution" ) ) {
-			this._worker.initialize( {
-				useKeywordDistribution: configuration.useKeywordDistribution,
-			} ).then( () => this.analyzePaper( state.paper ) );
+		if ( ! isEqual( prevConfiguration, configuration ) ) {
+			this._worker.initialize( configuration ).then( () => {
+				if ( state.configuration.isRelatedKeyphrase ) {
+					console.log( "analyzing related keyphrase..." );
```
Comment (igorschoester) [inline examples/webpack/src/redux/utils/StoreSubscriber.js]:
```suggestion
```

**Comment by nataliashitova** [comment]
Acceptance:
- [ ] Focus keyphrase
- [ ] Focus keyphrase cornerstone content
- [ ] Focus keyphrase taxonomy
- [ ] Related keyphrase
- [ ] Related keyphrase cornerstone content
- [ ] Related keyphrase taxonomy

**Comment by nataliashitova** [comment]
The new example is awesome! I am very happy we can now use it.
Small suggestion for the future: would be nice to make `Is cornerstone` and `Is taxonomy` mutually exclusive, meaning that only three possible combinations are available
- ` Is cornerstone` OFF, `Is taxonomy` OFF
- `Is cornerstone` ON, Is `taxonomy` OFF
- `Is cornerstone` OFF, Is `taxonomy` ON

The fourth possibility (`Is cornerstone` ON, `Is taxonomy` ON) is now available and calls the taxonomy assessor, but it would be nice to make it more explicit and force the `Is cornerstone` toggle to OFF if `Is taxonomy` toggle is turned ON and vice versa.

**Pull request merged**